src/Microsoft.AspNet.Mvc.Core/ApplicationModels/DefaultControllerModelBuilder.cs

```diff
@@ -58,6 +59,17 @@ public ControllerModel BuildControllerModel([NotNull] TypeInfo typeInfo)
                 }
             }
 
+            var controllerType = typeInfo.AsType();
+            foreach (var propertyInfo in controllerType.GetProperties(BindingFlags.Public | BindingFlags.Instance))
```

Comment:
This should be using `PropertyHelper` - this code already does things like leave out indexers.

Should we be looking at properties with a public getter?